### PR TITLE
feat(terra-release): add extra repo

### PR DESCRIPTION
**Pull request by madonuko**

### CODE REVIEW

File: anda/terra/release/terra-extra.repo [hunk 1]
```diff
@@ -0,0 +1,24 @@
+[terra]
```
Comment (lleyton) [inline anda/terra/release/terra-extra.repo]:
```suggestion
[terra-extra]
```

File: anda/terra/release/terra-extra.repo [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+metadata_expire=6h
+type=rpm
+gpgcheck=1
+gpgkey=https://repos.fyralabs.com/terra$releasever-extra/key.asc
+repo_gpgcheck=1
+enabled=1
+enabled_metadata=1
+countme=1
+
+[terra-source]
```
Comment (lleyton) [inline anda/terra/release/terra-extra.repo]: terra-extra-source

**Review by lleyton — CHANGES_REQUESTED** [review]
I wonder if we should decrease the priority of the extra repo, to prevent automatic installation of provides

**Review by lleyton — APPROVED** [review]

**Comment by raboneko** [comment]
## 💔 All backports failed

| Status | Branch | Result |
|:------:|:------:|:------|
|❌|f41|Backport failed because of merge conflicts|

### Manual backport
To create the backport manually run:
```
backport --pr 2400
```

### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sqren/backport) and see the [Github Action logs](https://github.com/terrapkg/packages/actions/runs/11550098498) for details

**Pull request merged**